examples/example-wasm/src/styles/TaskItem.module.css

```diff
@@ -0,0 +1,70 @@
+.task {
+  width: 300px;
+  min-width: 250px;
+  height: 300px;
```

Comment:
It's likely a personal preference of mine, but I'd like this to simply take the height of the content and width of the container.